tests/integration/trace/test_zipkin_trace.py

```diff
@@ -99,12 +97,15 @@ def test_zipkin_trace(trace_server):
 
     hostport = 'localhost:%d' % trace_server.port
 
-    response = yield tchannel.request(hostport).send(InMemStream(endpoint),
-                                                     InMemStream(hostport),
-                                                     InMemStream(),
-                                                     traceflag=True)
```

Comment:
because the old way to send request I put traceflag=True in the send call. So it will submit span for tracing.